src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue

```diff
@@ -58,9 +84,28 @@ export default {
     }
   },
   computed: {
-    validator() {
-      return this.validators[this.transaction.value.validator_address] || false
+    getValidators() {
```

Comment:
i think usually we name computer properties the thing that's returned - not like the name of a function. but just a little note / thought.

Reply:
The problem here is that there is already a `validators` as a prop

Reply:
ah ok. i didn't see that. still, should the prop be `allValidators` maybe?

i find it weird to have a computer property named like a function. but maybe just me!